src/utils/fetchClient.ts

```diff
@@ -13,6 +13,7 @@ type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
 function request<T>(
   url: string,
   method: RequestMethod = 'GET',
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   data: any = null, // we can send any data to the server
```

Comment:
Avoid disabling eslint rules without a good reason. If you need to disable a rule, provide a comment explaining why.